Code-the-Dream-School/dd-prac-team1-front
# Dev/terms PRA-124

## One Line Description
Added Terms of Service button in footer and .pdf file with terms

## Requirements

_What this code should do_
provide legal warning and inform users of terms of app usage

## Notes

_Any additional info_

## Test Steps
click on the Terms of Service button in the footer and see the pdf file opening in a new tab in browser

_Instructions on how to test the changes_

## UI/UX Outcome

_Attach a screenshot here_
![Olivier_Terms-pdf](https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/109550788/f3c578b1-a6c0-45fd-b375-a4f9cda25f4a)

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
